### PR TITLE
Add minor specification text corresponding to what has been implemented.

### PR DESCRIPTION
**Pull request by HansOlsson**

### CODE REVIEW

**Comment by HansOlsson** [comment]
@henrikt-ma check that ok, merge, and then we can change MCP-0029 into non-draft.

File: chapters/annotations.tex [hunk 1]
```diff
@@ -2323,6 +2323,12 @@ \subsection{Licensing}\label{licensing}
 \end{lstlisting}
 \end{example}
 
+\section{License Texts}\label{license-texts}
```
Comment (henrikt-ma) [inline chapters/annotations.tex:2326]: Restore air around `\section` commands:
```suggestion

\section{License Texts}\label{license-texts}

```

File: chapters/annotations.tex [hunk 2]
```diff
@@ -2323,6 +2323,12 @@ \subsection{Licensing}\label{licensing}
...
+For a top-level class the {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the class.
+The annotation may give a scalar value or an array of values.
+The license files are human-readable files containing the license conditions.
+External functions may also contain this annotation (with similar contents), see \cref{annotations-for-external-libraries-and-include-files}.
+When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.
```
Comment (henrikt-ma) [inline chapters/annotations.tex:2331]: Restore air around `\section` commands:
```suggestion
When generating a distributable version for a translated model the license files for used functions and top-level classes should be included.

```

File: chapters/functions.tex [hunk 3]
```diff
@@ -2398,6 +2398,11 @@ \subsection{Annotations for External Functions}\label{annotations-for-external-l
   {\lstinline!annotation(SourceDirectory="modelica:/ModelicaLibraryName/Resources/Source")!}\annotationindex{SourceDirectory}, gives the location for source files.
   The preceding one is the default and need not be specified; but another location could be specified by using an URI name for the source directory, see \cref{external-resources}.
   It is not specified how they are built.
+\item
+  The {\lstinline!annotation(License="modelica:/ModelicaLibraryName/Resources/Licenses/MyLicense.txt")!}\annotationindex{License}, gives the license files for the function.
```
Comment (henrikt-ma) [inline chapters/functions.tex:2402]: Would it make sense to give another example of what the URI for an external function could look like, showing that it doesn't have to be the same as the dedicated place for the top-level class?

**Review by henrikt-ma — COMMENTED** [review]
Only minor remarks, basically good for merging into MCP branch.

File: chapters/functions.tex (hunk 3 above)
Comment (HansOlsson) [reply chapters/functions.tex:2402]: I'm not sure - I understand many projects try to be restrictive for the location of licenses, so I wouldn't want to suggest that.

**Pull request merged**